packages/test/test-service-load/src/runner.ts

```diff
@@ -420,6 +420,7 @@ function scheduleOffline(
 		if (container.connectionState !== ConnectionState.Connected && !container.closed) {
```

Comment:
will !container.close be false for a disposed container? or put another way is a disposed container also closed?

Reply:
This bool handles both closed and disposed